app/views/scenarios/index.html.erb

```diff
@@ -21,7 +21,7 @@
         <% @scenarios.each do |scenario| %>
           <tr>
             <td>
-              <%= scenario_label(scenario, content_tag(:i, '', class: 'glyphicon glyphicon-font')) %>
+              <%= scenario_label(scenario, icon(scenario.icon)) %>
```

Comment:
@cantino I just put in a patch in the view to test `if scenario.icon` exist or not otherwise use the gear icon. I like the gear icon. I think its generic and good enough to put on the top.